server/src/graphql/mutations/destroyApp.ts

```diff
@@ -54,6 +54,22 @@ export const destroyApp: MutationResolvers['destroyApp'] = async (
   // Link to GH issue : https://github.com/prisma/migrate/issues/249
 
   // We delete the app itself
+  const githubMetaToDelete = await prisma.app
+    .findUnique({
+      where: {
+        id: appId,
+      },
+    })
+    .AppMetaGithub();
+
+  if (githubMetaToDelete) {
+    await prisma.appMetaGithub.delete({
```

Comment:
is there a way to do cascading deletion in prisma?

Reply:
@pradel the issue mentioned above is still active and doesn't seem to be cleanly fixed.